### ISSUE TITLE
Delete ground connections?

### ISSUE THREAD

**Issue by SarahBlau** [SarahBlau/BME590-Flashlight-Project#13]
If we're doing a polygon pour for our ground node, can we delete the lines that connect grounds to each other so the PCB routing looks cleaner?

**Comment by mlp6** [comment]
Yes... well, effectively, you aren't going to delete them, but you are going to connect them  to the polygon trace that is the ground plane.  Are you doing a one- or two-sided board?

**Comment by SarahBlau** [comment]
I'm not really sure actually - is a board with all the components on one side and a ground node on the other considered one-sided or two-sided?

**Comment by mlp6** [comment]
If it has conducting copper on both sides (like the `GND` plane on one side), then it is two-sided!  A one-sided board has no conductive copper at all on one side.

**Comment by SarahBlau** [comment]
I think it will be one sided in that case, since Brinnae suggested putting the ground plane on the same side as all the components.
